app/src/main/java/com/vero/tasky/agenda/data/local/entities/AttendeeEntity.kt

```diff
@@ -16,7 +16,8 @@ import androidx.room.PrimaryKey
     ]
 )
 data class AttendeeEntity(
-    @PrimaryKey(autoGenerate = false)
+    @PrimaryKey(autoGenerate = true)
+    val id: Int? = null,
```

Comment:
If im not mistaken, you could set it to be 0 by default

And that will make it so the id is not nullable (Room wont allow id 0, so it will automatically increment)